### PR TITLE
feat(dependsOn for items): enable item-to-item dependency

### PR DESCRIPTION
**Pull request by alghanor**
dependsOn type for refunded and canceled items

TM-4219

### CODE REVIEW

File: lib/v1/transactions/embedded/reference/depends.js [hunk 1]
```diff
@@ -30,12 +32,12 @@ module.exports = {
       }),
       default: null
     },
-    custom_id: {
```
Comment (qtotuan) [inline lib/v1/transactions/embedded/reference/depends.js]: Just clarified with Robert L.
Client_id is not the same as custom_id.
Client ID ist the  ID for idempotency. Custom ID is "Bon Nummer" or "[Irgendeine] Nummer".
The schema should have both. Does that make sense?

Comment (alghanor) [reply lib/v1/transactions/embedded/reference/depends.js]: both - that makes sense the most

**Comment by qtotuan** [comment]
Ah sorry, should have squashed the commits.

**Pull request merged**